### ISSUE TITLE
Better Latex support

### ISSUE THREAD

**Issue by jose-a-sa** [Jason-Rev/vscode-spell-checker#167]
VSCode has an awesome extension named Latex-Workshop by James-Yu.

In his GitHub readme.md it reads that we may add
```
"cSpell.ignoreRegExpList": [
  "\\\\\\w*(\\[.*?\\])?(\\{.*?\\})?",
  "\\$.+?\\$"
]
```
to vscode-spell-checker settings, but I was hoping that we could get a update with a dictionary for TeX commands.

**Comment by Jason3S** [comment]
Is there a good repository of sample LaTex files? It helps a lot to have good samples.

**Comment by Jason3S** [comment]
There is still more work to do for full LaTeX support, but I believe progress has been made.

**Comment by Jason3S** [comment]
This is the config file used for [LaTeX](https://raw.githubusercontent.com/Jason3S/cspell-dicts/master/latex/cspell-ext.json)

**Comment by felixlinker** [comment]
I experienced that spell checking does not work inside arguments. For example, inside a `\textit` command the text does not get spell-checked.

**Comment by Jason3S** [comment]
@felixlinker you are right. It skips all arguments except for the following commands: title, color, section, subsection, footnote, chapter, part, caption, emph.

The reason is that the command arguments were the primary cause of false positive spelling errors.

If you have a list of commands that should be added, please let me know.

**Comment by felixlinker** [comment]
It might be worth adding  all `\text[a-z]+` like commands to the list.
